### ISSUE TITLE
Uncaught ReferenceError: error is not defined

### ISSUE THREAD

**Issue by eightBitter** [atom/tree-view#512]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.1
**System**: Microsoft Windows 7 Enterprise 
**Thrown From**: [tree-view](https://github.com/atom/tree-view) package, v0.176.0

### Stack Trace

Uncaught ReferenceError: error is not defined

```
At C:\Users\jtshelby\AppData\Local\atom\app-1.0.1\resources\app.asar\node_modules\tree-view\lib\tree-view.js:829

ReferenceError: error is not defined
    at exit (C:\Users\jtshelby\AppData\Local\atom\app-1.0.1\resources\app.asar\node_modules\tree-view\lib\tree-view.js:829:60)
    at triggerExitCallback (C:\Users\jtshelby\AppData\Local\atom\app-1.0.1\resources\app.asar\src\buffered-process.js:213:47)
    at ChildProcess.<anonymous> (C:\Users\jtshelby\AppData\Local\atom\app-1.0.1\resources\app.asar\src\buffered-process.js:235:18)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:169:7)
    at Process.ChildProcess._handle.onexit (child_process.js:1057:12)
```

### Commands

```
```

### Config

```json
{}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

**Comment by izuzak** [comment]
cc @kevinsawicki -- seems related to https://github.com/atom/tree-view/pull/507.

**Comment by kevinsawicki** [comment]
Thanks for reporting this, will get a release out shortly with it addressed

**Comment by kevinsawicki** [comment]
The fix for this was just released in Atom 1.0.2
